Kiosk/Auction Listings/ListingsViewModel.swift

```diff
@@ -267,7 +267,16 @@ class ListingsViewModel: NSObject, ListingsViewModelType {
 // MARK: - Sorting Functions
 
 func leastBidsSort(lhs: SaleArtwork, _ rhs: SaleArtwork) -> Bool {
-    return (lhs.bidCount ?? 0) < (rhs.bidCount ?? 0)
+    switch (lhs.highestBidCents.hasValue, rhs.highestBidCents.hasValue) {
```

Comment:
I'm a big fan of this style of logic -- `swtich` on all the data required to make a decision